media/css/pebbles/components/_buttons-download.scss

```diff
@@ -143,15 +143,6 @@ ul.download-list {
     display: block !important;
 }
 
-// Add another rule for old browsers where :not isn't supported. If this is
-// mixed with the rule above, the entire rule will be invalid on those legacy
-// environments and some tests will fail.
-.osx:not(.pre-mavericks) .download-button .os_osx,
```

Comment:
I'm not seeing download buttons for any version of OSX. Assume it's because of the removal of this line?